<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectAccessedByFunctionWindow</class>
 <widget class="QDialog" name="ObjectAccessedByFunctionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Objects Accessed By Functions</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="selectedItemsListWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QTableView" name="objectsTableView">
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="QTreeWidget" name="callPathTreeWidget">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="indentation">
       <number>0</number>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <column>
       <property name="text">
        <string>DSO</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Function</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>File and Line</string>
       </property>
      </column>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exportToPdfPushButton">
       <property name="text">
        <string>Export to PDF</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showCacheUsagePushButton">
       <property name="text">
        <string>Show Cache Usage</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
